docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,461 @@
+** storage_connection
+** storage_container (defaults to Event Hub name if not defined)
+** consumer_group
+
+
+[[azure_config_options]]
+===== Configuration options
+
+NOTE: All Event Hubs options are common to both basic and advanced
+configurations, with the following exceptions. The basic configuration uses
+`event_hub_connections` to support multiple connections. The advanced
+configuration uses `event_hubs` and `event_hub_connection` (singular).
+
+[[azure_event_hubs]]
+===== `event_hubs`
+* Value type is <<array,array>>
+* No default value
+* Ignored for basic and command line configuration
+* Required for advanced configuration 
+
+Defines the per Event Hubs configuration for the <<adv-config-sample,advanced configuration>>. 
+
+The advanced configuration uses `event_hub_connection` instead of `event_hub_connections`. 
+The `event_hub_connection` option is defined per Event Hub.
+
+[[azure_event_hub_connections]]
+===== `event_hub_connections`
+* Value type is <<array,array>>
+* No default value
+* Required for basic and command line configuration
+* Ignored for advanced configuration
+
+List of connection strings that identifies the Event Hubs to be read. Connection
+strings include the EntityPath for the Event Hub. 
+
+
+[[azure_checkpoint_interval]]
+===== `checkpoint_interval`
+* Value type is <<number,number>>
+* Default value is `5` seconds
+* Set to `0` to disable.
+  
+Interval in seconds to write checkpoints during batch processing. Checkpoints
+tell {ls} where to resume processing after a restart. Checkpoints are
+automatically written at the end of each batch, regardless of this setting. 
+
+Writing checkpoints too frequently can slow down processing unnecessarily. 
+
+
+[[azure_consumer_group]]
+===== `consumer_group`
+* Value type is <<string,string>>
+* Default value is `$Default` 
+
+Consumer group used to read the Event Hub(s). Create a consumer group
+specifically for Logstash. Then ensure that all instances of Logstash use that
+consumer group so that they can work together properly.
+
+
+[[azure_decorate_events]]
+===== `decorate_events`
+
+* Value type is <<boolean,boolean>>
+* Default value is `false`
+
+Adds metadata about the Event Hub, including `Event Hub name`, `consumer_group`,
+`processor_host`, `partition`, `offset`, `sequence`, `timestamp`, and `event_size`.  
+
+
+[[azure_initial_position]]
+===== `initial_position`
+* Value type is <<string,string>>
+* Valid arguments are `beginning`, `end`, `look_back`
+* Default value is `beginning`
+
+When first reading from an Event Hub, start from this position: 
+
+* `beginning` reads all pre-existing events in the Event Hub 
+* `end` does not read any pre-existing events in the Event Hub 
+* `look_back` reads `end` minus a number of seconds worth of pre-existing events.
+You control the number of seconds using the `initial_position_look_back` option.
+
+Note: If `storage_connection` is set, the `initial_position` value is used only
```

Comment:
Need to write NOTE not Note if you want this to render in a note box.

Reply:
Thanks for looking.  In this case, I wanted to call out the info, but not with as much emphasis as the note box.